app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format."
+                         " Expected format: 'mv source destination'")
```

Comment:
There is a missing closing parenthesis in the `raise ValueError` statement. Ensure that the parentheses are balanced.